build.ps1

```diff
@@ -933,28 +937,37 @@ function Build-SourceKitLSP($Arch)
 function Build-Installer()
 {
   # Currently fails due to _InternalSwiftScan paths
-  # Build-WiXProject toolchain.wixproj -Properties @{
+  # Build-WiXProject toolchain.wixproj -Arch $HostArch -Properties @{
   #   DEVTOOLS_ROOT = "$ToolchainInstallRoot\";
   #   TOOLCHAIN_ROOT = "$ToolchainInstallRoot\";
   # }
 
-  # TODO: The XCTest depends on the architecture
-  # Build-WiXProject sdk.wixproj -Properties @{
-  #   PLATFORM_ROOT = "$PlatformInstallRoot\";
-  #   SDK_ROOT = "$SDKInstallRoot\";
-  #   SWIFT_SOURCE_DIR = "$SourceCache\swift\";
-  # }
+  foreach ($Arch in $SDKArchs)
+  {
+    # WiX v3 does not support ARM64
+    if ($Arch -eq $ArchARM64)
+    {
+      continue
+    }
```

Comment:
We still can package the ARM64 SDK as it is architecture agnostic.

Reply:
Hmm, plausibly.  I thought I had packaged up the ARM64 SDK previously for installation on X64.

Reply:
```
candle.exe : error CNDL0264: The parameter 'arch' is missing or has an invalid value arm64.  Possible values are x86, x64, or ia64. [S:\SourceCache\swift-installer-scripts\platforms\Windows\runtime.wixproj]

Reply:
Might've used a newer snapshot of WiX.  Migrating to 4.0 is going to require some work.